Extensions/Settings/Dnn.PersonaBar.Extensions/Components/CreateModuleController.cs

```diff
@@ -234,7 +234,7 @@ private int CreateModuleFromManifest(CreateModuleDto createModuleDto, out string
             try
             {
                 var folder = PathUtils.Instance.RemoveTrailingSlash(GetSourceFolder(createModuleDto));
-                var manifest = Path.Combine(Globals.ApplicationMapPath, "~/DesktopModules/" + folder + "/" + createModuleDto.Manifest);
+                var manifest = Path.Combine(Globals.ApplicationMapPath, "DesktopModules/" + folder + "/" + createModuleDto.Manifest);
```

Comment:
Hi @zyhfish, may I suggest the following:
```suggestion
                var manifest = Path.Combine(Globals.ApplicationMapPath, "DesktopModules", folder, createModuleDto.Manifest);
```

Reply:
applied, thx @daguiler !